pytest_html/resources/style.css

```diff
@@ -28,6 +29,25 @@ table {
 	border-collapse: collapse;
 }
 
+@media (prefers-color-scheme: dark) {
```

Comment:
What does the browser support look like for this annotation? Will this break older browsers?

Reply:
Seems pretty recent: https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme

How will older browsers handle this?

Reply:
general browser support: https://caniuse.com/mdn-css_at-rules_media
special browser support: https://caniuse.com/prefers-color-scheme

old browsers will not be broken. they will simply ignore it

Reply:
Nice! Thanks!